src/scene/render.js

```diff
@@ -0,0 +1,21 @@
+import { EventHandler } from '../core/event-handler.js';
+
+function Render() {
+    EventHandler.call(this);
+    this._meshes = null;
+}
+
+Render.prototype = Object.create(EventHandler.prototype);
+Render.prototype.constructor = Render;
+
+Object.defineProperty(Render.prototype, 'meshes', {
```

Comment:
is this missing generateWireframe?

Reply:
this is function called by model component
render component implement this directly in the component